src/Microsoft.Crank.PullRequestBot/Program.cs

```diff
@@ -108,7 +108,10 @@ static async Task<int> Main(string[] args)
                     "Any additional arguments to pass through to crank."),
                 new Option<string>(
                     "--config",
-                    "The path to a configuration file.") { IsRequired = true }
+                    "The path to a configuration file.") { IsRequired = true },
+                new Option<int>(
+                    "--age",
```

Comment:
Are there any docs that need updating with the new option?

Reply:
Yes :/